docs/UserGuide.md

```diff
@@ -168,56 +170,72 @@ Examples:
 
 TuteeTally's payment management commands are designed to streamline the financial interactions between tutors and students, ensuring accuracy and transparency. Below are detailed explanations of each command's purpose and benefits:
 
-### Adding Payments: `addpayment`
+### Adding a Payment: `addpayment`
 
 Record new payments easily with the `addpayment` command, keeping track of what each student owes. This command simplifies the maintenance of financial records, ensuring you never overlook an outstanding payment.
 
-**Format:** `addpayment -id {ID} -payment {AMOUNT}`
-**Example:** `addpayment -id 000001 -payment 150` or `addpayment -id 1 -payment 150` would add a payment of 150 to the student whose ID is '#000001' if it exists. <br>
+Format: `addpayment -id {ID} -payment {AMOUNT}`
+
+Example: 
+* `addpayment -id 000001 -payment 150` or `addpayment -id 1 -payment 150` would add a payment of 150 to the student whose ID is '#000001' if it exists. <br>
 If the student has no outstanding payments, a 'No payment owed' message will be displayed. <br>
-Following the command, the display will update to show "Payment owed: $150".
+Following the command, the display will update to show "Payment owed: $150". <br>
 After entering the command, the interface will update as shown below:
 
 ![Add Payment Update Display](images/payment/add_payment.png) _The display showing "Payment owed: $150" after adding a payment._
 
 <box type="info" seamless>
+
+**Note:** <br>
 * All payments added will be rounded off to the nearest $0.01.
+
 </box>
 
-### Marking Payments as Paid: `markpayment`
+### Marking Payment of the student: `markpayment`
+
+The `markpayment` command allows you to update the status of a student's payment to reflect payments that have been made. This feature helps maintain a clear record of completed transactions, building trust and transparency. <br>
 
-The `markpayment` command allows you to update the status of a student's payment to reflect payments that have been made. This feature helps maintain a clear record of completed transactions, building trust and transparency.
+Format: `markpayment -id {ID} -payment {AMOUNT}`
 
-**Format:** `markpayment -id {ID} -payment {AMOUNT}`
-**Example:** `markpayment -id 000001 -payment 100` or `markpayment -id 000001 -payment 100` whose ID is '#000001' if it exists. <br>
+Example:
+* `markpayment -id 000001 -payment 100` or `markpayment -id 000001 -payment 100` whose ID is '#000001' if it exists. <br>
```

Comment:
Both of these seems to be the same,

did u mean -id 1 for the first and -id 000001 for the second?

Reply:
oops will fix